br/pkg/version/version_test.go

```diff
@@ -536,3 +536,28 @@ Check Table Before Drop: false`
 	require.Error(t, err)
 	require.Regexp(t, "mock failure$", err.Error())
 }
+
+func TestFetchVersionWithCommitID(t *testing.T) {
+	db, mock, err := sqlmock.New()
+	require.NoError(t, err)
+
+	tidbVersionWithCommitInReleaseVersion := `Release Version: bcf61918
```

Comment:
please add a positive case for normal `release version`

Reply:
thanks for commend, the positive case has been added